### DISCUSSION TITLE
Calendar mode -view missing event "dots" after last update

### DISCUSSION THREAD

**Discussion by slayeri** [totaldebug/atomic-calendar-revive#711] [Support] [answered]
Hi,

After updating to the latest version, my calendar mode is missing the small dots indicating that date has some event stored on it.

Before the update, there was a small dot (an eye-symbol or something) indicating if the date had some event, but now there isn't any more. I attached a screencap where I've highlighted a date that has an event, and you can see there is no indication of that on the calendar view.

Is this expected behavior, or is something broken on my end? I really liked the feature where I could glance in the future to see upcoming events.

![image](https://user-images.githubusercontent.com/72884855/174567502-0054b7b2-b500-4420-b021-8f1cad7008c4.png)

**Comment by marksie1988** [comment, marked as answer]
You would need to configure the entities with the `icon` setting to show an icon on calendar view. If this is not set, no icon will appear.

**Reply by slayeri** [reply]
Thanks :)
